.github/workflows/nightly.yaml

```diff
@@ -166,6 +166,8 @@ jobs:
       - uses: ./.github/actions/clean-runner
       - name: Build image
         run: |
+          # enable username namespaces
```

Comment:
Actually not needed here.

Reply:
It is needed at line 201

Reply:
See https://github.com/project-zot/zot/actions/runs/12821167569/job/35767842486

Reply:
fixed